hotsos/defs/scenarios/lxd/lxcfs_deadlock.yaml

```diff
@@ -27,3 +35,11 @@ conclusions:
         threads and eventually a kernel crash because of hung tasks. The bug
         is fixed in LXD 5.10. For more information see
         https://github.com/lxc/lxcfs/issues/471.
+  fuse_deadlock:
+    decision: fuse_out_of_threads_error
```

Comment:
apologies i know I suggested this but actually can you add the following:

```
  fuse_deadlock:
    decision:
      - has_lxd_version_5_9
      - fuse_out_of_threads_error
```

i.e. put the lxd version check back in so that if lxd is upgraded we dont raise the issue anymore

Reply:
ack done thanks